prover/src/config.rs

```diff
@@ -35,8 +39,10 @@ pub struct ProverServiceConfig {
     pub enable_test_provider: bool,
     #[serde(default)]
     pub enable_federated_jwks: bool,
-    #[serde(default)]
-    pub disable_iat_in_past_check: bool,
+    #[serde(default = "default_true")]
```

Comment:
Nit: Why do you need `default = default_true` instead of `default`?

Reply:
default value is false i'm pretty sure.  

that was the reason i used negative language because i wanted the default behavior when it isn't set to be set the desired conservative one.

but since we want the checks enabled by default, then default val has to be true.